tests/perf-test/SearchPage.perf-test.js

```diff
@@ -177,16 +169,16 @@ test.skip('[Search Page] should search in options list', async () => {
         .then(() => measurePerformance(<SearchPageWrapper navigation={navigation} />, {scenario, runs}));
 });
 
-test.skip('[Search Page] should click on list item', async () => {
+test('[Search Page] should click on list item', async () => {
     const {triggerTransitionEnd, addListener} = TestHelper.createAddListenerMock();
 
     const scenario = async () => {
         await screen.findByTestId('SearchPage');
         const input = screen.getByTestId('options-selector-input');
 
-        fireEvent.changeText(input, mockedPersonalDetails['6'].login);
+        fireEvent.changeText(input, mockedPersonalDetails['4'].login);
```

Comment:
any particular reason we have changed the personal details?

Reply:
no reason, just remainder after some tweaking in tests. It shouldn't affect the flakiness